Project2015To2017.Core/Definition/AssemblyAttributes.cs

```diff
@@ -121,7 +136,7 @@ public string GetAttribute(Type attributeType)
 
 			//Make the assumption that it just has a single string argument
 			//because all of the attributes we currently look for do have
-			return att?.ArgumentList.Arguments.Single().ToString().Trim('"');
+			return att?.ArgumentList.Arguments.Single().ToString().TrimStart('@').Trim('"');
```

Comment:
I don't believe this change will achieve anything. The `@` at the start e.g. `@"c:\something"` is just for reading the typed string, the resultant string in memory will be the same as if somebody had escaped slashes directly `"c:\\something"`.